### PR TITLE
Publish site

### PR DESCRIPTION
**Pull request by Eomm**
TODOS

- [x] home page
- [x] upload articoles from dev to
- [ ] configure google analytics (could you do it @fox1t )
- [x] the site is published even with an opened PR
- [ ] no use PNG transparent

### CODE REVIEW

**Comment by Eomm** [comment]
I have asked support to a friend of mine to get some resize of the logo to complete the homepage

What articles should I migrate? https://dev.to/eomm @fox1t

**Comment by fox1t** [comment]
I don't understand why the site is published already. I think we should just merge this PR and then iterate over it.

**Review by fox1t — APPROVED** [review]

File: about/index.md [hunk 1]
```diff
@@ -7,6 +7,29 @@ eleventyNavigation:
   order: 3
 ---
 
-I am a person that writes stuff.
+This blog talks about software development, focusing on the Backend things made with Node.js!
+We will cover topics such as:
 
-This blog template is based on [eleventy-high-performance-blog](https://www.industrialempathy.com/posts/eleventy-high-performance-blog/) which itself is based on [eleventy-base-blog](https://github.com/11ty/eleventy-base-blog).
+- how to...
+- security concerns
```
Comment (fox1t) [inline about/index.md:14]: I really like this about page! It is so inspiring!

**Comment by Eomm** [comment]
Supersedes by https://hashnode.com/616b2988441bd36daa955d1f/dashboard/analytics

**Pull request closed without merging**